tests/checks/performance/performance_report_test.py

```diff
@@ -142,8 +142,7 @@ def test_condition_min_score_not_passed(iris_split_dataset_and_model):
     # Assert
     assert_that(result, has_items(
         equal_condition_result(is_pass=False,
-                               details=re.compile(r'Scores that did not pass the threshold:<br>'
-                                                  r'\[\{\'Class\':'),
+                               details=re.compile(r'Found metrics with subceeding scores:\n'),
```

Comment:
which metrics and what score?

Reply:
Too complicated result to write regex for